tests/autoyast/validate_salt_formula.pm

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2020 SUSE LLC
```

Comment:
Not sure where did you get this header, could you please use one starting with:
```
# SUSE's openQA tests
...
```

Reply:
added

Reply:
you cannot mix/edit license as far as I know, I think the ones starting with that text I pointed you are more updated (only contains 1 year), you need to pick the whole paragraph. The one that you used is an old one where the year is updated once the file is touched.

Reply:
(C) ->  ©